cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -408,6 +416,13 @@ func main() {
 		log.SetOutput(ioutil.Discard)
 	}
 
+	if *fdRlimit == 0 {
+		*fdRlimit = expectedFDs
+	}
+	if err := limits.SetupFDLimits(*fdRlimit); err != nil {
+		log.Println(err)
```

Comment:
Please use the `logging` library's `logging.Infof` here.

Reply:
Done.